unittest/gunit/CMakeLists.txt

```diff
@@ -247,6 +250,8 @@ ADD_DEFINITIONS(-DMYSQL_SERVER)
 ADD_DEFINITIONS(-DERRMSG_DIR="${PROJECT_BINARY_DIR}/sql/share")
 ADD_DEFINITIONS(-DDATA_DIR="${CMAKE_CURRENT_BINARY_DIR}")
 
+INCLUDE(${MYSQL_CMAKE_SCRIPT_DIR}/compile_flags.cmake)
```

Comment:
Shouldn't this be included before `MY_CHECK_CXX_COMPILER_FLAG("-Wformat-overflow" HAVE_FORMAT_OVERFLOW)`?

Reply:
`MY_CHECK_CXX_COMPILER_FLAG` is defined in `./CMakeLists.txt`, not in `compile_flags.cmake`